docs/docs/Integrations/Nvidia/integrations-nvidia-nim-wsl2.md

```diff
@@ -0,0 +1,33 @@
+---
+title:  Integrate NVIDIA NIMs with Langflow
+slug: /integrations-nvidia-ingest-wsl2
+---
+
+Connect **Langflow** with **NVIDIA NIM** on an RTX Windows system with [Windows Subsystem for Linux 2 (WSL2)](https://learn.microsoft.com/en-us/windows/wsl/install) installed.
+
+[NVIDIA NIM](https://docs.nvidia.com/nim/index.html) provides containers to self-host GPU-accelerated inferencing microservices.
+This example deploys the `mistral-nemo-12b-instruct` NIM on an **RTX Windows system** with **WSL2** and uses it as a model component in **Langflow**.
```

Comment:
I'm not opposed to the bolding, but I'm also not sure that it is standard style (though I'm not familiar with the langflow docs). Just dropping a note to reconsider it.